### PR TITLE
Update locales-hr-HR.xml

### PR DESCRIPTION
**Pull request by tvrbanec**
Updated and refined translation.

### CODE REVIEW

File: locales-hr-HR.xml [hunk 1]
```diff
@@ -69,13 +69,13 @@
...
-    <term name="ordinal-11">th</term>
-    <term name="ordinal-12">th</term>
-    <term name="ordinal-13">th</term>
+    <term name="ordinal"></term>
+    <term name="ordinal-01"></term>
+    <term name="ordinal-02"></term>
+    <term name="ordinal-03"></term>
+    <term name="ordinal-11"></term>
+    <term name="ordinal-12"></term>
+    <term name="ordinal-13"></term>
```
Comment (rmzelle) [inline locales-hr-HR.xml]: If there are no ordinal suffixes in Croatian, you can just replace

```
    <!-- ORDINALS -->
    <term name="ordinal"></term>
    <term name="ordinal-01"></term>
    <term name="ordinal-02"></term>
    <term name="ordinal-03"></term>
    <term name="ordinal-11"></term>
    <term name="ordinal-12"></term>
    <term name="ordinal-13"></term>
```

with

```
    <!-- ORDINALS -->
    <term name="ordinal"></term>
```

**Comment by rmzelle** [comment]
Can you make the change I requested (use edit button at https://github.com/tvrbanec/locales/blob/patch-2/locales-hr-HR.xml)? Otherwise it all looks good!

**Comment by tvrbanec** [comment]
A s far as I know :-), there are no ordinals in Croatian...

**Comment by rmzelle** [comment]
> As far as I know :-), there are no ordinal suffixes in Croatian...

Wikipedia lists Croatian as one of the languages that uses an ordinal period (like German): https://en.wikipedia.org/wiki/Ordinal_indicator#Representation_as_period

Does that sound familiar?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**